taskfiles/deps/main.yaml

```diff
@@ -214,8 +214,8 @@ tasks:
             - "-DFMT_DOC=OFF"
             - "-DFMT_TEST=OFF"
           LIB_NAME: "{{.G_FMT_LIB_NAME}}"
-          TARBALL_SHA256: "b06ca3130158c625848f3fb7418f235155a4d389b2abc3a6245fb01cb0eb1e01"
-          TARBALL_URL: "https://github.com/fmtlib/fmt/archive/refs/tags/8.0.1.tar.gz"
+          TARBALL_SHA256: "ede1b6b42188163a3f2e0f25ad5c0637eca564bd8df74d02e31a311dd6b37ad8"
+          TARBALL_URL: "https://github.com/fmtlib/fmt/archive/refs/tags/10.0.0.tar.gz"
```

Comment:
Shall we use the latest 10.x version (10.2.1)?

Also, why not update to the latest spdlog and fmtlib versions?

Reply:
I initially used the earliest release compatible with musllinux, but I'm okay with switching to the latest version.